crates/graphql-composition/tests/composition/namespaced_directives_basic/subgraphs/alerts.graphql

```diff
@@ -0,0 +1,25 @@
+extend schema @link(url: "https://example.com/kafka/v2.3")
+
+type Alert @key(fields: "id") {
+  id: ID!
+  message: String!
+  severity: Severity!
+}
+
+enum Severity {
+  FINE
+  OK
+  GOOD
+}
+
+type Mutation {
+  createAlert(message: String!, severity: Severity!): Alert @kafka__post(topic: "alerts")
```

Comment:
Am I missing something or should this directive be in the federated graph somewhere?

Reply:
No, as long as it hasn't been registered as an extension directive (see https://linear.app/grafbase/document/extensions-sdl-ffd3e6c1fe3b). For these, there will be a transformation, they are not passed into the federated graph verbatim.